cime_config/machines/config_machines.xml

```diff
@@ -651,10 +669,35 @@
     <environment_variables>
       <env name="NETCDF_PATH">$ENV{NETCDF_DIR}</env>
       <env name="PNETCDF_PATH">$ENV{PNETCDF_DIR}</env>
+      <env name="NTASKS_PER_GPU"> </env>
+      <env name="GPU_BIND_ARGS"> </env>
     </environment_variables>
     <environment_variables compiler="amdclang">
       <env name="LD_LIBRARY_PATH">$ENV{CRAY_LIBSCI_DIR}/amd/4.0/x86_64/lib:$ENV{LD_LIBRARY_PATH}</env>
     </environment_variables>
+    <environment_variables compiler=".*gpu.*">
+      <env name="NTASKS_PER_GPU">--ntasks-per-gpu=2</env>
+      <env name="GPU_BIND_ARGS">--gpu-bind=closest</env>
+    </environment_variables>
+    <environment_variables compiler=".*gpu.*" DEBUG="TRUE">
+      <env name="AMD_LOG_LEVEL">10</env>
+      <env name="CRAY_ACC_DEBUG">3</env>
+    </environment_variables>
+    <environment_variables compiler="gnugpu">
+      <env name="PNETCDF_HINTS">romio_cb_read=disable</env>
+      <env name="MPICH_GPU_SUPPORT_ENABLED">1</env>
+      <env name="MPIR_CVAR_GPU_EAGER_DEVICE_MEM">0</env>
+    </environment_variables>
+    <environment_variables compiler="crayclanggpu">
+      <env name="PNETCDF_HINTS">romio_cb_read=disable</env>
+      <env name="MPICH_GPU_SUPPORT_ENABLED">0</env>
```

Comment:
So, we are enabling MPICH_GPU_SUPPORT for amdclanggpu but not for crayclangpu?
I recall Scream has issues with GPU-aware MPI. Is that the reason or something else?